flit_core/flit_core/common.py

```diff
@@ -398,7 +398,7 @@ def write_metadata_file(self, fp):
     def supports_py2(self):
         """Return True if Requires-Python indicates Python 2 support."""
         for part in (self.requires_python or "").split(","):
-            if re.search(r"^\s*(>\s*(=\s*)?)?[3-9]", part):
+            if re.search(r"^\s*(>=?|~=|===?)?\s*?[3-9]", part):
```

Comment:
```suggestion
            if re.search(r"^\s*(>=?|~=|===?)?\s*[3-9]", part):
```
Was there a reason to have the `?` following `\s*`? I don't think it achieves anything ("zero or one times zero or more spaces").

Reply:
Right, this `?` doesn't achieve much, thanks for spotting it!